basyx.submodelservice/basyx.submodelservice-core/src/test/java/org/eclipse/digitaltwin/basyx/submodelservice/SubmodelServiceSuite.java

```diff
@@ -362,6 +362,23 @@ public void updateNonFileSME() {
 		assertEquals(newProperty, updatedProperty);
 	}
 
+	@Test
+    public void updateNonNestedSME() {
+        Submodel technicalSubmodel = DummySubmodelFactory.createTechnicalDataSubmodel();
+        SubmodelService submodelService = getSubmodelService(technicalSubmodel);
+        Property property = createDummyProperty("test123");
+
+        submodelService.createSubmodelElement(property);
+
+        String idShortPathRootSME = "test123";
+
+        Property newProperty = SubmodelServiceHelper.createDummyProperty("test123", "arbitraryValue", DataTypeDefXsd.STRING);
+        submodelService.updateSubmodelElement(idShortPathRootSME, newProperty);
+
+        Property updatedProperty = (Property) submodelService.getSubmodelElement(idShortPathRootSME);
```

Comment:
Please rename updatedProperty to actualUpdatedProperty

Reply:
done